src/app/modules/main/wallet_section/send/module.nim

```diff
@@ -46,6 +48,8 @@ proc newModule*(
   result.view = newView(result)
   result.controller = controller.newController(result, events, walletAccountService, networkService, currencyService, transactionService)
   result.moduleLoaded = false
+  result.senderCurrentAccountIndex = 0
+  result.receiveCurrentAccountIndex = 0
```

Comment:
Could you add a todo that we should create a dedicated module for that?

Reply:
fir the receive modal? sure!